### DISCUSSION TITLE
The Future of Arcticons (Dramatic title but it's just some updates regarding my personal life)

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#412] [Announcements]
Arcticons is growing and growing, getting more contributors and requests, it costs more time for me to maintain. I like doing it but don't always have time because of my internship, work and school stuff. Thankfully because of some great people here, other things are easier. I make long days, so when I get home in the evening I don't always have time to check what's happening here, so most stuff happens in the weekends, when I spend a lot of Saturdays and Sundays making icons, curating/ fixing them, updating other stuff and making releases while also trying to maintain my social life.

With many things happening here on a daily basis, it was flooding my main mailbox (should have created a separate mail for that a long time ago, oof). But yea, that makes things hard to keep track of, especially now that a lot of issues are more than just icons.

So yeah, stuffs happening here, and it's great. But things are most likely gonna be on a lower pace than it has been in the past.

Thanks everyone, for their contributions and support. <3

**Comment by Aman9das** [comment]
Just make someone a collaborator if you believe they can curate the icons.. for less load and for the bus factor

And you needn't worry too much because it's open source :^)

**Reply by Donnnno** [reply]
Curating is something I'd rather do myself, but thanks!
